sql-plugin/src/main/330until340/scala/org/apache/spark/sql/rapids/shims/RapidsErrorUtils.scala

```diff
@@ -61,7 +62,7 @@ object RapidsErrorUtils extends RapidsErrorUtilsFor330plus {
     )
   }
 
-  def throwSparkDateTimeException(infOrNan: String): Unit = {
+  def throwSparkDateTimeException(infOrNan: String): SparkDateTimeException = {
```

Comment:
The method name implies it throws an exception but it doesn't.  Probably should be `sparkDateTimeException`.

Reply:
updated in [2421439](https://github.com/NVIDIA/spark-rapids/pull/6956/commits/2421439eb5cb9e6bfefc3d2280f0d273f2042638)